web/hooks.go

```diff
@@ -26,35 +28,56 @@ func EnableHook(w http.ResponseWriter, r *http.Request) {
 	user := vars["user"]
 	repo := vars["repo"]
 	validator := strings.ToLower(vars["validator"])
-	cfg := config.Read()
-	cookiename := cfg.Settings.CookieName
-	sessionid, err := r.Cookie(cookiename)
+	ut, err := getSessionOrRedirect(w, r)
+	if err != nil {
+		return
```

Comment:
The error message is potentially not logged. `getSessionOrRedirect` logs in one error case and does not in another.

Reply:
`getSessionOrRedirect` logs the error only if the user has a session cookie set and the server doesn't have a session with that ID. This is the second case where the error is `Invalid session found in cookie`. The second case, `No session cookie found` isn't really an error, it just occurs when a user goes to a page that requires login and has no session cookie. I don't think it's necessary to log this, but I can add it if you want.